stable/cluster-autoscaler/values.yaml

```diff
@@ -49,6 +49,7 @@ image:
   repository: k8s.gcr.io/cluster-autoscaler
   tag: v1.12.0
   pullPolicy: IfNotPresent
+  pullSecrets: []
```

Comment:
Can you add some comments to reference on how this to be used eg. https://github.com/helm/charts/blob/master/stable/searchlight/values.yaml#L17-L23